dss-api.yml

```diff
@@ -766,14 +766,10 @@ paths:
 
                   ```
 
-                  --2769baffc4f24cbc83ced26aa0c2f712
                   Content-Disposition: form-data; name="foo"
-
                   bar
```

Comment:
Should `bar` still be here?

Reply:
Yes - this block of code contains a header + body, and `bar` is the body

Reply:
Ah, I didn't understand it.  Since this is documentation, could this be made clearer?

Reply:
For context: the request body shown has multiple header+body frames stacked together. This is perfectly acceptable (see these links from Mozilla: [HTTP/2 Frames section of this  page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Messages) and [multipart/form-data section of this page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#multipartform-data)). 

Reviewing the second link, I believe the `----------------2769baffc4f24cbc83ced26aa0c2f712` was intended to be a marker between frames, and was just improperly formatted.